### PR TITLE
Add Xarray-Spatial Classification Tutorial Notebook

### PR DESCRIPTION
**Pull request by brendancol**

### CODE REVIEW

File: tutorials/xarray-spatial_classification-methods.ipynb [hunk 1]
```diff
@@ -0,0 +1,372 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "837aff2c",
+   "metadata": {},
+   "source": [
+    "In this tutorial, you'll learn how to use different classification methods with [xarray-spatial](https://github.com/makepath/xarray-spatial) to classify the terrain elevation levels of the Grand Canyon.\n",
+    "\n",
+    "Geo-spatial data [classification](http://wiki.gis.com/wiki/index.php/Classification) algorithms assign groups of data to categories, or classes, for further processing. Classification is used when grouping data points into classes for different colored areas on a choropleth map, for example. [xarray-spatial](https://github.com/makepath/xarray-spatial) is a raster analysis tool and contains different classification methods.\n",
```
Comment (TomAugspurger) [inline tutorials/xarray-spatial_classification-methods.ipynb:18]:
```suggestion
    "Geo-spatial data [classification](http://wiki.gis.com/wiki/index.php/Classification) algorithms assign groups of data to categories, or classes, for further processing. Classification is used to group data points into classes for different colored areas on a choropleth map, for example. [xarray-spatial](https://github.com/makepath/xarray-spatial) is a raster analysis tool and contains different classification methods.\n",
```

File: tutorials/xarray-spatial_classification-methods.ipynb [hunk 2]
```diff
@@ -0,0 +1,372 @@
...
+   "execution_count": null,
+   "id": "1c5d7b05",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from datashader.transfer_functions import shade\n",
+    "import matplotlib.pyplot as plt\n",
+    "from xrspatial.classify import natural_breaks\n",
+    "\n",
+    "natural_breaks_agg = natural_breaks(img_arr, num_sample=20000, k=15)\n",
```
Comment (TomAugspurger) [inline tutorials/xarray-spatial_classification-methods.ipynb:198]: A bit of guidance for choosing these parameters would be helpful I think.

Comment (TomAugspurger) [reply tutorials/xarray-spatial_classification-methods.ipynb:198]: Also, thoughts on mentioning what `natural_breaks_agg` is, before plotting it? Just something brief like 

   Like most methods in xarray-spatial `natural_breaks` returned an `xarray.DataArray`. It contains integers representing ...

File: tutorials/xarray-spatial_classification-methods.ipynb [hunk 3]
```diff
@@ -0,0 +1,372 @@
...
+   "source": [
+    "#### Classify with `equal_interval()`"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "baed3e1b",
+   "metadata": {},
+   "source": [
+    "To classify data into sets based on intervals of equal width, use the [equal interval](https://xarray-spatial.org/reference/_autosummary/xrspatial.classify.equal_interval.html) function. The [equal interval classification](http://wiki.gis.com/wiki/index.php/Equal_Interval_classification) is useful in cases where you want to emphasize the amount of an attribute value relative to the other values."
```
Comment (TomAugspurger) [inline tutorials/xarray-spatial_classification-methods.ipynb:224]: I'm not sure I understand what "amount of an attribute value" means. Maybe rephrase it / expand on it?

**Review by TomAugspurger — COMMENTED** [review]
Looks great, thanks.

Can you merge all adjacent markdown cells? Makes for fewer button presses when executing the notebook.

File: tutorials/xarray-spatial_classification-methods.ipynb (hunk 3 above)
Comment (brendancol) [reply tutorials/xarray-spatial_classification-methods.ipynb:224]: @TomAugspurger hey yes

- [ ] rephrase "amount of an attribute value"
- [ ] explain natural breaks agg
- [ ] merge adjacent cells
- [ ] guidance on choosing classificaion parameters

**Comment by TomAugspurger** [comment]
Thanks!
